src/io.rs

```diff
@@ -0,0 +1,94 @@
+//! A module for writing TFRecords, Tensorflow's preferred on-disk data format.
+//!
+//! See the [tensorflow docs](https://www.tensorflow.org/api_guides/python/python_io#tfrecords-format-details) for details of this format.
+
+extern crate byteorder;
+extern crate crc32c;
+
+use std::io;
+use std::io::prelude::*;
```

Comment:
Please don't use wildcards in imports.  It makes it difficult to tell where imported symbols came from and which traits are in use.

Reply:
Done.